src/Marqeta.Core.Abstractions/Generated.cs

```diff
@@ -15293,6 +15293,21 @@ public partial class Webhook_response_model_paginated_response
         public int? Start_index { get; set; }
     
     
+    }
+    
+    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "10.0.23.0 (Newtonsoft.Json v11.0.0.0)")]
+    public partial class Currency_conversion_network 
+    {
+        [Newtonsoft.Json.JsonProperty("original_currency_code", Required = Newtonsoft.Json.Required.Default, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public string Original_currency_code { get; set; }
+    
+        [Newtonsoft.Json.JsonProperty("conversion_rate", Required = Newtonsoft.Json.Required.Default, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public double? Conversion_rate { get; set; }
```

Comment:
Looks like these are coming through as "double?"s is that ok?

Reply:
we do double to decimal conversion in other areas, so should be fine